common/views/components/SubNavigation/SubNavigation.tsx

```diff
@@ -33,6 +32,12 @@ type Props = {
   hasDivider?: boolean;
 };
 
+const Divider = styled.div`
```

Comment:
Mega nitp so take it or leave it, but I'd be tempted to call it something else to differentiate it from the component